### ISSUE TITLE
`renameTo` doesn't rename annotations in type parameters

### ISSUE THREAD

**Issue by steveluscher** [facebook/jscodeshift#199]
I expected this to rename the type annotation in the type parameter, but it didn't.

```
/**
 * const A = require('ModuleA');
 * 
 * class Foo<A> {
 *  render(foo: A): A {
 *    return A;
 *   }
 * }
 */
export default function transformer(file, api) {
  const j = api.jscodeshift;
  return j(file.source)
    .find(j.VariableDeclarator, {id: {name: 'A'}})
    .forEach(path => { j(path).renameTo('B'); })
    .toSource();
}
```

Repro: http://astexplorer.net/#/gist/02d7f52e10dd9c7d803f433635931369/abc53a869b55bc61aa1ded6062d166d655b7c2cc

**Comment by steveluscher** [comment]
This is because a `TypeParameter` is not an `Identifier`.

https://github.com/facebook/jscodeshift/blob/e4fdb0201cbe0d411178d5c264dcad4224363013/src/collections/VariableDeclarator.js#L87

**Comment by bgw** [comment]
In case anyone runs into this, it's actually because `TypeParameter` wasn't being traversed. It does have an `Identifier` child. It's fixed by benjamn/ast-types#253, so facebook/jscodeshift#239 (a version bump) should fix this in jscodeshift.
